src/middlewares/auth/utils/getResourceBusinessIdInUpdate.js

```diff
@@ -0,0 +1,14 @@
+const { BusinessClass } = require("../../../models");
+
+const getResourceBusinessIdInUpdate = async (req, res) => {
+  let { businessClassId } = req.params;
+  console.log(businessClassId);
+
+  let businessClass = await BusinessClass.findById(businessClassId);
+  if (!businessClass) return false;
+  if (businessClass) {
+    console.log(businessClass.businessId);
```

Comment:
remove console log. console should only be there in catch block or where we are handling error